src/core/lib/iomgr/poller/eventmanager_libuv.cc

```diff
@@ -0,0 +1,77 @@
+/*
+ *
+ * Copyright 2019 gRPC authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+#include <grpc/support/port_platform.h>
+
+#include "src/core/lib/iomgr/poller/eventmanager_libuv.h"
+
+#include <grpc/support/time.h>
+
+grpc::experimental::LibuvEventManager::Options::Options() : num_workers_(-1) {}
+grpc::experimental::LibuvEventManager::Options::Options(int num_workers)
+    : num_workers_(num_workers) {}
+
+grpc::experimental::LibuvEventManager::LibuvEventManager(const Options& options)
+    : options_(options), should_stop_(0), shutdown_refcount_(0) {
```

Comment:
Since this is the only constructor, can you move the constant initializers to the .h file?

Reply:
It looks that we cannot use constant initializers here since implicit copy constructor is unavailable in grpc_core::Atomic: https://source.cloud.google.com/results/invocations/668417cb-4678-477b-82a9-f0166db51bec/targets/grpc%2Fcore%2Fpull_request%2Flinux%2Fgrpc_bazel_build/log

Reply:
So, you can use uniform initialiation in those cases; we use it in some of our codes. `grpc_core::Atomic<int> x{23};` rather than `grpc_core::Atomic<int> x = 23;`